srcpkgs/LibreCAD/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'LibreCAD'
 pkgname=LibreCAD
 version=2.1.3
-revision=5
+revision=6
```

Comment:
If this is only fixing the build, we don't have to revbump.

Reply:
Got it.